src/coreclr/pal/tests/palsuite/threading/NamedMutex/test1/namedmutex.cpp

```diff
@@ -84,11 +84,12 @@ char *BuildName(const char *testName, char *buffer, const char *prefix0, const c
     if (g_isStress)
     {
         // Append the test name so that tests can run in parallel
-        nameLength += test_snprintf(&buffer[nameLength], nameLength, "%s", testName);
+        nameLength += test_snprintf(&buffer[nameLength], strlen(testName) + 1, "%s", testName);
```

Comment:
I wonder what's the benefit of computing the size instead of using the remainder of the buffer. To me, that's how the snprintf is supposed to be used - you pass it the space left in the buffer and it will ensure it won't write out of the buffer.

Reply:
Currently the callers doesn't pass the _size_, so I computed it in-place. I agree that difference approach is more suitable. I will pass the sizes from the call-sites.

Reply:
Turned out we are using the same constant in this function. The different sized array was `char name[257];`, which we don't pass to BuildName.